tests/integration/records/records_schema_v1_schema.json

```diff
@@ -1,294 +1,58 @@
 {
```

Comment:
I don't understand what's going on in this file but I'll trust you that it's all good